scrapeGithub/database.py

```diff
@@ -0,0 +1,56 @@
+import sqlite3
+import pdb
+
+def connectToDatabase(database):
+    conn = sqlite3.connect(database, timeout=10)
+    createTable = """
+    CREATE TABLE IF NOT EXISTS "urls_to_be_scraped" (
+	"id"	INTEGER PRIMARY KEY AUTOINCREMENT,
+	"url"	TEXT NOT NULL UNIQUE,
+	"isScraped"	INTEGER DEFAULT 0,
+    "isValidReadme" INTEGER DEFAULT 1,
```

Comment:
Add a comment explaining when a readme can be invalid

Reply:
Not sure, already have added that in TODO.